### DISCUSSION TITLE
Tiling with lines of constant thickness

### DISCUSSION THREAD

**Discussion by hshackle** [cduck/hyperbolic#15] [Q&A] [answered]
Hi! Thanks a lot for creating this package; I've found it very helpful. I have a question regarding the `tiles.ipynb` example. The thickness of the geodesic lines that are drawn vary with the distance from the boundary. Is there any way to modify this code so that the thickness is constant, or is this not easily doable with the current code?

**Comment by cduck** [comment, marked as answer]
This is possible.

`hwidth` sets the width of the line in hyperbolic space.  This is why the thickness appears to vary when projected into the Poincare disk.  You can simply draw the lines normally by replacing `hwidth` and `fill` with `stroke_width`, `stroke`, and `fill='none'`.  Change
```python
d.draw(tile, hwidth=0.02, fill='white')
```
into
```python
d.draw(tile, stroke_width=0.02, stroke='white', fill='none')
```
